specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefits.json

```diff
@@ -0,0 +1,1002 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "usageDate": {
+          "description": "Date corresponding to the utilization record. If the grain of data is monthly, it will be first day of month.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "IncludedQuantityUtilizationSummary": {
+      "description": "Included Quantity utilization summary resource.",
+      "x-ms-discriminator-value": "IncludedQuantity",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummary"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IncludedQuantityUtilizationSummaryProperties",
+          "title": "Benefit utilization summary for benefit kind Included Quantity."
+        }
+      }
+    },
+    "IncludedQuantityUtilizationSummaryProperties": {
+      "type": "object",
+      "description": "Included Quantity utilization summary properties.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummaryProperties"
+        }
+      ],
+      "properties": {
+        "utilizationPercentage": {
+          "description": "This is the utilized percentage for the benefit ID.",
+          "type": "number",
+          "format": "decimal",
+          "readOnly": true
+        }
+      }
+    },
+    "BenefitUtilizationResource": {
+      "description": "The Resource model definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      }
+    },
+    "SavingsPlanUtilizationSummary": {
+      "description": "Savings plan utilization summary resource.",
+      "x-ms-discriminator-value": "SavingsPlan",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummary"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SavingsPlanUtilizationSummaryProperties",
+          "title": "Benefit utilization Summary for benefit kind savings plan."
+        }
+      }
+    },
+    "SavingsPlanUtilizationSummaryProperties": {
+      "type": "object",
+      "description": "Savings plan utilization summary properties.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationSummaryProperties"
+        }
+      ],
+      "properties": {
+        "avgUtilizationPercentage": {
```

Comment:
>avgUtilizationPercentage [](http://example.com/codeflow?start=9&length=24)

I'm not sure if we encourage abbreviating 'average' as 'avg'. I think its probably actually discouraged, although you can likely find examples both ways.

Reply:
Not addressing this suggestion. Keeping as 'avg'